lib/midi.coffee

```diff
@@ -8,19 +8,36 @@ L = utils = midi = midi_in = ___ = null
   midi = rtMidi
   ___ = utils.____ '[midi]'
 
+# Parse method copied from https://github.com/hhromic/midi-utils-js/blob/master/midiparser.js#L418
```

Comment:
Since we have no license currently, I see no problem with making the whole thing MIT licensed. In the license I believe the only stipulation is that the license is included anywhere the code goes.